src/pages/home.html

```diff
@@ -1,126 +1,121 @@
----
-title: 'Open Web Advocacy'
-metaDesc: 'A group of software engineers from all over the world who have come together to advocate for the future of the open web'
-permalink: '/'
-layout: 'layouts/home.njk'
-isHome: true
----
+--- title: 'Open Web Advocacy' metaDesc: 'A group of software engineers from all over the world who have come together to advocate for the future of the open web' permalink: '/' layout: 'layouts/home.njk' isHome: true ---
```

Comment:
It appears this formatting change of the front matter is breaking 11ty.
```sh
1:13:11 PM: $ npm run production
1:13:11 PM: > 11ty-base@1.0.0 production
1:13:11 PM: > NODE_ENV=production eleventy
1:13:12 PM: [11ty] Problem writing Eleventy templates: (more in DEBUG output)
1:13:12 PM: [11ty] > Having trouble reading front matter from template ./src/pages/home.html
1:13:12 PM: `TemplateContentFrontMatterError` was thrown
1:13:12 PM: [11ty] > gray-matter engine "title: 'Open Web Advocacy' metaDesc: 'A group of software engineers from all over the world who have come together to advocate for the future of the open web' permalink: '/' layout: 'layouts/home.njk' isHome: true ---" is not registered
1:13:12 PM: `Error` was thrown:
1:13:12 PM: [11ty]     Error: gray-matter engine "title: 'Open Web Advocacy' metaDesc: 'A group of software engineers from all over the world who have come together to advocate for the future of the open web' permalink: '/' layout: 'layouts/home.njk' isHome: true ---" is not registered
1:13:12 PM:         at module.exports (/opt/build/repo/node_modules/gray-matter/lib/engine.js:6:11)
1:13:12 PM:         at module.exports (/opt/build/repo/node_modules/gray-matter/lib/parse.js:8:18)
1:13:12 PM:         at parseMatter (/opt/build/repo/node_modules/gray-matter/index.js:109:17)
1:13:12 PM:         at matter (/opt/build/repo/node_modules/gray-matter/index.js:50:10)
1:13:12 PM:         at Template.read (/opt/build/repo/node_modules/@11ty/eleventy/src/TemplateContent.js:112:14)
1:13:12 PM:         at async Template.getFrontMatterData (/opt/build/repo/node_modules/@11ty/eleventy/src/TemplateContent.js:207:7)
1:13:12 PM:         at async Template.getData (/opt/build/repo/node_modules/@11ty/eleventy/src/Template.js:404:29)
1:13:12 PM:         at async Template.getTemplateMapEntries (/opt/build/repo/node_modules/@11ty/eleventy/src/Template.js:1066:33)
1:13:12 PM:         at async TemplateMap.add (/opt/build/repo/node_modules/@11ty/eleventy/src/TemplateMap.js:59:21)
1:13:12 PM:         at async Promise.all (index 6)
1:13:12 PM: [11ty] Wrote 0 files in 0.05 seconds (v1.0.0)
1:13:12 PM: ​
1:13:12 PM: ────────────────────────────────────────────────────────────────
1:13:12 PM:   "build.command" failed                                        
1:13:12 PM: ────────────────────────────────────────────────────────────────
```

So if that can be restored, the build should pass and the deploy preview should work.

----

As an aside, it might be worth exploring adding Prettier to our repo.  This could / would help avoid formatting / styling changes from showing up in PRs, which say in a case like this, might help it easier to distinguish formatting from content changes when reviewing the rest of the PR.

Anyway, just thinking out loud.

Reply:
Hmm the fixing the frontmatter is easy enough but seeing how the indentation was changed throughout the entire file, it may be better to just redo this? I can find some time before the end of the day.

I'd also suggest switching to youtube-nocookie.com for the embed. It's still not GDPR compliant but it's a slight improvement, at least.

@brucelawson ?

Reply:
I'll be honest, I have no idea what 11ty is, or does. I'd be happy to switch to youtube-nocookie.com

Reply:
The part at the top of the file (between `---`) is called 'front matter'. Basically a place where you can define some stuff, usually(?) in yml format. I think your editor decided it could just put all of that stuff on one line, which breaks the yml parser :)

The same thing appears to have happened with the indentation throughout the file (your editor changed everything from 2 to 4).

Which editor are you using? Does it support .editorconfig? That should automatically prevent this from happening.

Reply:
I think GitHub allows owners of forks to let repo owners push to forked branches now, so unless anyone objects, I'm happy to update this branch to at least get the build passing @brucelawson.  Others can review approve for the content as well.

Just want to help get this through if I can help in any way!  👍